docs/health/health_design/design.rst

```diff
@@ -34,27 +90,29 @@ Here is the pyATS Health Check yaml. It's almost same with `Blitz`! There are a
                 processor: both
                 # `function` can be found from Genie Feature Browser
                 # Please find the link to the page from bottom of this section
-                function: get_platform_cpu_load
+                function: health_cpu
```

Comment:
why is there a new health specific API?

Reply:
same with https://github.com/CiscoTestAutomation/genie/pull/34#discussion_r666936546